### PR TITLE
Add ability to set commissioner node ID and update logging.

### PR DESCRIPTION
**Pull request by krypton36**
Summary:
- Add ability to set commissioner Node ID in darwin-framework-tool.
- Update logging statements.

Testing:
- Tested Ability to change/set controller Node ID by passing in additional parameter.
- Test alpha, beta and gamma controllers.

### CODE REVIEW

**Review by liamgonyea — APPROVED** [review]
Changes look good to me

File: examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm [hunk 1]
```diff
@@ -137,9 +137,13 @@
     ipk = [gNocSigner getIPK];
 
     constexpr const char * identities[] = { kIdentityAlpha, kIdentityBeta, kIdentityGamma };
+    std::string commissionerName = mCommissionerName.HasValue() ? mCommissionerName.Value() : kIdentityAlpha;
     for (size_t i = 0; i < ArraySize(identities); ++i) {
         auto controllerParams = [[MTRDeviceControllerStartupParams alloc] initWithIPK:ipk fabricID:@(i + 1) nocSigner:gNocSigner];
 
+        if (commissionerName.compare(identities[i]) == 0 && mCommissionerNodeId.HasValue()) {
```
Comment (bzbarsky-apple) [inline examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm:144]: So commissioner-nodeid will only work in non-interactive mode, basically?  Do we need to do "subscribe with a non-default node id"?

File: examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h [hunk 2]
```diff
@@ -38,6 +38,7 @@ class CHIPCommandBridge : public Command {
         : Command(commandName)
     {
         AddArgument("commissioner-name", &mCommissionerName);
+        AddArgument("commissioner-nodeId", 0, UINT64_MAX, &mCommissionerNodeId);
```
Comment (bzbarsky-apple) [inline examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h:41]: This should probably have a docstring explaining the constraints on this (e.g. that it only works in non-interactive mode for now), right?

File: examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm [hunk 3]
```diff
@@ -75,6 +75,11 @@ - (void)handleQueryImageForNodeID:(NSNumber * _Nonnull)nodeID
         _selectedCandidate.userConsentNeeded = _userConsentNeeded;
         NSLog(@"User Consent Needed: %@", _selectedCandidate.userConsentNeeded);
     }
+    if (_delayedActionTime && _selectedCandidate.status.integerValue == MTROtaSoftwareUpdateProviderOTAQueryStatusBusy) {
```
Comment (bzbarsky-apple) [inline examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm:78]: This does not seem related to the commissioner node id?

File: examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm (hunk 1 above)
Comment (krypton36) [reply examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm:144]: Yes, to a certain extent it will not work in interactive mode if the initial params are not provided and will only work on a single controller. For the use case, which is OTA Update, it should be sufficient. At a later time, if we need to set the controller node IDs, we can scope out the work, because we would need to change the "stack startup" portion of the code to start controllers on demand.

File: examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h (hunk 2 above)
Comment (krypton36) [reply examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h:41]: I will add docstring. This does working in Interactive mode if pass in the arguments when starting interactive mode, but it only works for a single controller, because of how the stack starts with initializing the controllers.

File: examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm (hunk 3 above)
Comment (krypton36) [reply examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm:78]: Yes, apologies. It is part of the changes necessary for cert. Would you like me to create another PR for this?

Comment (bzbarsky-apple) [reply examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm:78]: Yes, please!

**Comment by bzbarsky-apple** [comment]
@krypton36 So with the changes here, once you run with `--commissioner-nodeid` that node id will get persisted and will keep being used until you use a different `--commissioner-nodeid` value or reset the darwin-framework-tool storage, right?

**Comment by krypton36** [comment]
> @krypton36 So with the changes here, once you run with `--commissioner-nodeid` that node id will get persisted and will keep being used until you use a different `--commissioner-nodeid` value or reset the darwin-framework-tool storage, right?

This is exactly the behavior of this parameter.

File: examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h [hunk 4]
```diff
@@ -38,6 +38,9 @@ class CHIPCommandBridge : public Command {
         : Command(commandName)
     {
         AddArgument("commissioner-name", &mCommissionerName);
+        AddArgument("commissioner-nodeId", 0, UINT64_MAX, &mCommissionerNodeId,
+            "Sets the commisser node ID of the given "
+            "comssioner-name. Interactive mode will only set a single commissioner on the inital command.");
```
Comment (bzbarsky-apple) [inline examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h:43]:
```suggestion
            "commissioner-name. Interactive mode will only set a single commissioner on the inital command. "
            "The commissioner node ID will be persisted until a different one is specified.");
```

**Review by bzbarsky-apple — APPROVED** [review]
